app/imports/ui/pages/EditSmartContract.jsx

```diff
@@ -0,0 +1,128 @@
+import { Meteor } from 'meteor/meteor';
+import { withTracker } from 'meteor/react-meteor-data';
+import PropTypes from 'prop-types';
+import SimpleSchema2Bridge from 'uniforms-bridge-simple-schema-2';
+import { Stuffs } from '../../api/stuff/Stuff';
+
+const panes = [
+  { menuItem: 'Parties Involved', render: () => <Tab.Pane>
+    <TextField name='name'/>
+    <TextField name='email'/>
+    <TextField name='phoneNumber'/>
+    <SelectField name='role'/>
+    <SubmitField value='Save'/>
+  </Tab.Pane> },
+  { menuItem: 'Smart Contract Type', render: () => <Tab.Pane>
+    <NumField name='NumberOfTenets' decimal={false}/>
+    <SubmitField value='Save'/>
+  </Tab.Pane> },
+  { menuItem: 'Terms & Conditions', render: () => <Tab.Pane>
+    <p>Terms and conditions for smart contract with __ number of tenets. Cost, frequency of payment, parties involved.</p>
+    <SelectField name='stance'/>
+    <SubmitField value='Save'/>
+  </Tab.Pane> },
+  { menuItem: 'Payment Information', render: () => <Tab.Pane>
+    <TextField name='billingAddress'/>
+    <TextField name='bankAccountNumber'/>
+    <TextField name='bankAccountRoutingNumber'/>
+    <SubmitField value='Save'/>
+  </Tab.Pane> },
+  { menuItem: 'Party Review Status', render: () => <Tab.Pane>
+    <Table definition>
+      <Table.Header>
+        <Table.Row>
+          <Table.HeaderCell />
+          <Table.HeaderCell>Role</Table.HeaderCell>
+          <Table.HeaderCell>Terms & Conditions Approval Status</Table.HeaderCell>
+        </Table.Row>
+      </Table.Header>
+
+      <Table.Body>
+        <Table.Row>
+          <Table.Cell>John Doe</Table.Cell>
+          <Table.Cell>Tenet</Table.Cell>
+          <Table.Cell>Approved</Table.Cell>
+        </Table.Row>
+        <Table.Row>
+          <Table.Cell>Mary Jane</Table.Cell>
+          <Table.Cell>Tenet</Table.Cell>
+          <Table.Cell>Rejected</Table.Cell>
+        </Table.Row>
+        <Table.Row>
+          <Table.Cell>Naruto Uzumaki</Table.Cell>
+          <Table.Cell>Homeowner</Table.Cell>
+          <Table.Cell>Incomplete</Table.Cell>
+        </Table.Row>
+      </Table.Body>
+    </Table>
+  </Tab.Pane> },
+  { menuItem: 'Sign', render: () => <Tab.Pane>
+    <p>I hereby accept the terms and conditions outlined in this smart contract.</p>
+    <TextField name='signature'/>
+    <SubmitField value='Create smart contract!'/>
+  </Tab.Pane> },
+];
+
+const bridge = new SimpleSchema2Bridge(Stuffs.schema);
+
+/** Renders the Page for editing a single document. */
+class EditSmartContract extends React.Component {
+
+  // On successful submit, insert the data.
+  submit(data) {
+    const { name, email, phoneNumber, role, numberOfTenets, stance, billingAddress, bankAccountNumber, bankAccountRoutingNumber, signature, _id } = data;
+    Stuffs.collection.update(_id, { $set: { name, email, phoneNumber, role, numberOfTenets, stance, billingAddress, bankAccountNumber, bankAccountRoutingNumber, signature } }, (error) => (error ?
```

Comment:
remove billingAddress, bankAccountNumber, and bankAccountRoutingNumber

add in accountAddress